addons/mail/static/src/components/message/message.xml

```diff
@@ -114,75 +101,54 @@
                                     </Popover>
                                 </t>
                             </t>
-                            <div class="o_Message_commands o_Message_headerCommands" t-att-class="{ 'o-mobile': messaging.device.isMobile }">
-                                <t t-if="!message.isTemporary and ((message.message_type !== 'notification' and message.originThread and message.originThread.model === 'mail.channel') or !message.isTransient)">
-                                    <span class="o_Message_command o_Message_commandStar o_Message_headerCommand fa"
-                                        t-att-class="{
-                                            'fa-star': message.isStarred,
-                                            'fa-star-o': !message.isStarred,
-                                            'o-message-selected': isSelected,
-                                            'o-message-starred': message.isStarred,
-                                            'o-mobile': messaging.device.isMobile,
-                                        }" t-on-click="_onClickStar" title="Mark as Todo"
-                                    />
-                                </t>
-                                <t t-if="props.hasReplyIcon">
-                                    <span class="o_Message_command o_Message_commandReply o_Message_headerCommand fa fa-reply"
-                                        t-att-class="{
-                                            'o-message-selected': isSelected,
-                                            'o-mobile': messaging.device.isMobile,
-                                        }" t-on-click="_onClickReply" title="Reply"
-                                    />
-                                </t>
-                                <t t-if="props.hasMarkAsReadIcon">
-                                    <span class="o_Message_command o_Message_commandMarkAsRead o_Message_headerCommand fa fa-check"
-                                        t-att-class="{
-                                            'o-message-selected': isSelected,
-                                            'o-mobile': messaging.device.isMobile,
-                                        }" t-on-click="_onClickMarkAsRead" title="Mark as Read"
-                                    />
-                                </t>
-                            </div>
                         </div>
                     </t>
-                    <div class="o_Message_content" t-ref="content">
-                        <div class="o_Message_prettyBody" t-ref="prettyBody"/><!-- message.prettyBody is inserted here from _update() -->
-                        <t t-if="message.subtype_description and !message.isBodyEqualSubtypeDescription">
-                            <p t-esc="message.subtype_description"/>
+                    <div class="o_Message_contentWrapper">
```

Comment:
I don't think the wrapper should be around the content, it should probably be the whole message component instead.

Reply:
That doesn't allow us to place the actions relative to the message text as per the spec.

Reply:
Why it doesn't allow it? I'm not sure to understand which spec is referred to in that case.
We want the highlight specifically on the content and not on the whole message?

Reply:
see first slide of: https://docs.google.com/presentation/d/1WjrTC4YLn_ug8D8OdrGU1HHQe5iAb78U-rHZXrcy8g0/edit#slide=id.gb81872c42a_0_14

Maybe this mockup is not actually what we want? But having played with it a bit I think it makes for a more uniform experience across squashed and non-squashed messages. I'm open to changing to a version where we highlight the entire message instead of just the text and place actions relative to it though.